src/Sulu/Bundle/AdminBundle/Resources/js/components/MultiItemSelection/MultiItemSelection.js

```diff
@@ -21,11 +21,26 @@ type Props<T, U, V> = {|
     sortable: boolean,
 |};
 
-const ItemWrapper = ({children}: Object) => (
-    <li className={multiItemSelectionStyles.listElement}>
-        {children}
-    </li>
-);
+type ItemWrapperProps = {
+    children: Element<typeof Item>,
+    isDisabled: boolean,
```

Comment:
We are usually only using `disabled`.

Reply:
Ok, weird that the `SortableElement` does not support that... But it is not that bad, since that component is only used within this file.